Automattic/course
# Set wide template as default for courses

Fix issue [Add a template for the post type "Course"#6091](https://github.com/Automattic/sensei/issues/6091)

### Introduced change
- Set the wide template as default for courses


### Testing instructions
- Create a course
- Check if the template was selected by default.

<img width="491" alt="Screenshot 2022-11-22 at 18 34 04" src="https://user-images.githubusercontent.com/38718/203425803-ec451874-d4f0-4e46-8efc-e7570d65eaf4.png">
